Source/WebKit/Shared/Cocoa/SandboxExtensionCocoa.mm

```diff
@@ -81,8 +81,10 @@
     return byteCast<uint8_t>(m_token.span());
 }
 
-char* SandboxExtensionImpl::sandboxExtensionForType(const char* path, SandboxExtension::Type type, std::optional<audit_token_t> auditToken, OptionSet<SandboxExtension::Flags> flags)
+CString SandboxExtensionImpl::sandboxExtensionForType(const char* path, SandboxExtension::Type type, std::optional<audit_token_t> auditToken, OptionSet<SandboxExtension::Flags> flags)
 {
+    char* sandboxExtension = nullptr;
```

Comment:
I think we could use this:
```
auto sandboxExtension = [&] {
    switch (type) {
    case SandboxExtension::Type::ReadOnly:
        return std::unique_ptr<char, decltype(free)*>(sandbox_extension_issue_file(APP_SANDBOX_READ, path, extensionFlags), free);
         // ...
     }
});
```

Reply:
Ah, very nice, will fix.

Thanks for reviewing!